boa/src/syntax/parser/statement/mod.rs

```diff
@@ -214,22 +214,24 @@ where
 ///
 /// [spec]: https://tc39.es/ecma262/#prod-StatementList
 #[derive(Debug, Clone, Copy)]
-pub(super) struct StatementList {
+pub(super) struct StatementList<'a> {
     allow_yield: AllowYield,
     allow_await: AllowAwait,
     allow_return: AllowReturn,
     break_when_closingbraces: bool,
```

Comment:
Looks really good! Just a couple of minor things which might be nice to include. 'break_when_closingbraces' no-longer seems to actually reflect the new behaviour as any of the break nodes will trigger the check - can you either rename it or look to remove it entirely.

Reply:
I think that if we use `}` as a `break_node`, then we don't need this anymore, and can remove it from the structure.

Reply:
Ok, I took it out and everything appears to still be fine.